{NA <type 'exceptions.NameError'> "name 'NON_EXISTENT_FILENAME' is not defined"
 Traceback (most recent call last):
  File "regression-tests/NA_iter.py", line 4, in <module>
    for line in open(NON_EXISTENT_FILENAME):
}
{NA <type 'exceptions.TypeError'> 'coercing to Unicode: need string or buffer, NA found'
 Traceback (most recent call last):
  File "regression-tests/NA_iter.py", line 4, in <module>
    for line in open(NON_EXISTENT_FILENAME):
}
{NA_iternext | regression-tests/NA_iter.py <module>:4}
